modules/monitoring-common-terms.adoc

```diff
@@ -0,0 +1,122 @@
+
+
+custom resource (CR)::
+A CR is extension of the Kubernetes API. You can create custom resources.
+
+Elasticsearch::
+
+etcd::
+
+Federation endpoint::
+
+Fluentd::
+
+HAProxy::
+
+Image registry::
+
+Kubelets::
+Runs on nodes and reads the container manifests. Ensures that the defined containers have started and are running.
+
+Kubernetes API server::
+
+Kubernetes controller manager::
+Kubernetes controller manager governs the state of the cluster.
+
+Kubernetes scheduler::
+Kubernetes scheduler allocates pods to nodes.
+
+labels::
+You can use labels, which are key-value pairs, to organise and select subsets of objects, such as a pod.
+
+Metering::
+
+metrics::
+
+metric target::
+
+monitoring dashboards::
+
+monitoring stack::
+
+node::
+A worker machine in the {product-title} cluster. A node is either a virtual machine (VM) or a physical machine.
+
+NVIDIA GPU administration dashboard::
+
+Operator::
+The preferred method of packaging, deploying, and managing a Kubernetes application in an {product-title} cluster. An Operator takes human operational knowledge and encodes it into software that is packaged and shared with customers.
+
+{product-title} API server::
+
+{product-title} Controller Manager::
+
+Operator Lifecycle Manager (OLM)::
+
+Persistent storage::
+Stores the data even after the device is shut down. Kubernetes uses persistent volumes to store the application data
+
+Persistent volume claim (PVC)::
+You can use a PVC to mount a PersistentVolume into a Pod. You can access the storage without knowing the details of the cloud environment.
+
+pod::
+The pod is the smallest logical unit in Kubernetes. A pod contains one or more containers to run in a worker node.
+
+project::
+
+Prometheus::
+Prometheus is the monitoring system on which the OpenShift Container Platform monitoring stack is based. Prometheus is a time-series database and a rule evaluation engine for metrics. Prometheus sends alerts to Alertmanager for processing.
+
+Prometheus adapter::
+
+Prometheus Operator::
+The Prometheus Operator (PO) in the openshift-monitoring project creates, configures, and manages platform Prometheus instances and Alertmanager instances. It also automatically generates monitoring target configurations based on Kubernetes label queries.
+
+routing::
+
+Silences::
+A silence can be applied to an alert to prevent notifications from being sent when the conditions for an alert are true. You can mute an alert after the initial notification, while you work on resolving the underlying issue.
+
+storage::
+{product-title} supports many types of storage, both for on-premise and cloud providers. You can manage container storage for persistent and non-persistent data in an {product-title} cluster.
+
+Thanos Ruler::
```

Comment:
The Thanos Ruler is a rule evaluation engine for Prometheus that is deployed as a separate process. In OpenShift Container Platform, Thanos Ruler provides rule and alerting evaluation for the monitoring of user-defined projects.

Reply:
Addressed